components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -78,4 +81,72 @@ class ActionSheetTest: XCTestCase {
     }
     
   }
+
+  func testColorThemer() {
+    // Given
+    let titleTest = "Title"
+
+    // When
+    actionSheet.title = titleTest
+
+    // Then
+    if let title = getLabel(with: titleTest) {
+      XCTAssertEqual(title.textColor, .black)
+    } else {
+      XCTFail("No title label")
+    }
+
+    // Given
```

Comment:
Please split these into separate tests.

Reply:
Done

Reply:
Looks like there are still two different tests in here. If these should be the same test, please combine the related given, when, and then clauses.

Reply:
Done